WebUI/lims/AppDatabase.py

```diff
@@ -58,22 +80,72 @@ def executeSql(self, query, connection):
     # Get the replicates based upon if they are running (or not) with an upper limit for how many (default 1000)
     @staticmethod
     def getReplicates(request, running, limit = 1000):
-        SQL = """
-            SELECT id, filename, starttime, endtime, movement,
-                CASE WHEN endtime IS NULL THEN (now() - starttime) ELSE runningtime END AS runningtime
-            FROM v_replicates {} ORDER BY starttime DESC LIMIT %(limit)s"""
+        if running:
+            SQL = """
+                SELECT filename, starttime, movement,
+                    CASE WHEN endtime IS NULL THEN (now() - starttime) ELSE runningtime END AS runningtime
+                FROM v_replicates {} ORDER BY starttime DESC LIMIT %(limit)s"""
+        else:
+            SQL = """
+                SELECT id, filename, starttime, endtime, movement,
+                    CASE WHEN endtime IS NULL THEN (now() - starttime) ELSE runningtime END AS runningtime
+                FROM v_replicates {} ORDER BY starttime DESC LIMIT %(limit)s"""
```

Comment:
Why are these queries different?

Reply:
Because for different pages, they require different columns from database. For index page, it does not require replicates' ids and end time, but for last 100 replicates page, we need ids and end time based on our previous discussion.